core/templates/services/exploration-html-formatter.service.ts

```diff
@@ -78,94 +78,113 @@ export class ExplorationHtmlFormatterService {
       interactionCustomizationArgs: InteractionCustomizationArgs,
       parentHasLastAnswerProperty: boolean,
       labelForFocusTarget: string,
-      savedSolution: string | null): string {
-    var htmlInteractionId = this.camelCaseToHyphens.transform(interactionId);
-    var element = $('<oppia-interactive-' + htmlInteractionId + '>');
-
+      savedSolution: 'savedMemento()' | null
```

Comment:
Perhaps add unit tests for this function where you try injecting unsafe stuff in the focus target, saved solution, etc. -- anything that is used for crafting the final html string -- and make sure the correct behaviour happens (e.g. the right exceptions are thrown).

Reply:
Done.